src/general/storageclass/src.rego

```diff
@@ -20,7 +20,6 @@ storageclass_found(name) {
 }
 
 violation[{"msg": pvc_storageclass_badname_msg}] {
-  input.parameters.includeStorageClassesInMessage == true
```

Comment:
Please run 'make generate' to generate the constraint template file to include recent updates in src.rego.